Libplanet.Explorer.Executable/Program.cs

```diff
@@ -100,17 +100,22 @@ public static void Main(string[] args)
                         peers.Add(peer);
                     }
 
-                    await swarm.AddPeersAsync(
+                    await swarm.BootstrapAsync(
                         peers,
+                        5000,
+                        5000,
                         cancellationToken: cts.Token
                     );
 
-                    ImmutableHashSet<Address> trustedPeers =
-                        peers.Select(p => p.Address).ToImmutableHashSet();
+                    /*ImmutableHashSet<Address> trustedPeers =
+                        peers.Select(p => p.Address).ToImmutableHashSet();*/
+                    var trustedPeers = ImmutableHashSet<Address>.Empty;
+                    Console.WriteLine("**********Preload started");
```

Comment:
As it's a diagnostic log it should be written to [stderr] rather than stdout.

```suggestion
                    Console.Error.WriteLine("Starts preloading.");
```

[stderr]: https://en.wikipedia.org/wiki/Standard_streams#Standard_error_(stderr)